src/main/java/org/opentestsystem/shared/contentspecid/ContentSpecIdConverter.java

```diff
@@ -41,83 +45,92 @@ public ContentSpecId parse(String input) throws IdParseException {
         return parse(input, ContentSpecGrade.UNSPECIFIED);
     }
 
+
     public ContentSpecId parse(String input, ContentSpecGrade grade) throws IdParseException {
         if (getFormatType(input).isLegacy()) {
```

Comment:
Can we add documentation to this method including examples of `input`?  And maybe something around what is mandatory?